MaxText/tests/attention_test.py

```diff
@@ -560,48 +551,103 @@ def tpu_kernel_attention_helper(self, num_kv_heads):
         jax.numpy.allclose(mha_generic_output, mha_generic_flash_output, rtol=1e-01, atol=1e-01, equal_nan=False)
     )
 
+  @pytest.mark.tpu_only
+  def test_tpu_flash_attention_cp_and_ep(self):
+    # cp
+    self.tpu_flash_attention_cp_and_ep_helper(
+        ici_context_parallelism=4,  # use context parallelism of 4
```

Comment:
Nit: probably we could clean up those comments? I think config's name present its purpose? Similar to other comments bellow.